homeassistant/__main__.py

```diff
@@ -358,23 +309,13 @@ def main():
 
     args = get_arguments()
 
+    if args.script is not None:
+        from homeassistant import scripts
+        return scripts.run(args.script)
```

Comment:
No, that's why the script arg in main uses the remain option. Try it by
passing help before and after invoking the dB migrate script

On Sat, Jul 2, 2016, 22:38 Johann Kellerman <notifications@github.com>
wrote:

> In homeassistant/__main__.py
> <https://github.com/home-assistant/home-assistant/pull/2426#discussion_r69385902>
> :
>
> > @@ -358,23 +309,13 @@ def main():
> >
> >      args = get_arguments()
> >
> > +    if args.script is not None:
> > +        from homeassistant import scripts
> > +        return scripts.run(args.script)
>
> If there are two levels of 'argparse', here and in the script, wont
> duplicate --args be interpreted by the first argparse and never be passed
> to script.run?
>
> (i.e -c or --config in the script)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/2426/files/fd92c34a19241ec6d9f8c3cd19b7d3511fd4fc85#r69385902>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABYJ2iuC-a_PDkAVi_FktFK0pthL3PRSks5qR0rCgaJpZM4JDvUs>
> .
>